eng/tox/run_apistubgen.py

```diff
@@ -63,7 +64,7 @@ def get_package_wheel_path(pkg_root):
     if not pkg_path:
         pkg_path = args.target_package
 
-    cmds = ["apistubgen", "--pkg-path", pkg_path]
```

Comment:
Perhaps the wrong executable is being called in? This ensures that we are using the correct `site-packages` folder by invoking the module from directly.